<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessDialog</class>
 <widget class="QWidget" name="MessDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MessDialog</string>
  </property>
  <widget class="QLabel" name="Title_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>280</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>标题</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="DocBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WorkButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>查看</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
